README.md

```diff
@@ -0,0 +1,20 @@
+# Archivematica documentation
+
+[Archivematica](https://www.archivematica.org/en/) is a web- and standards-based, open-source application which allows your institution to preserve long-term access to trustworthy, authentic and reliable digital content.
+Our target users are archivists, librarians, and anyone working to preserve digital objects.
+
+You are free to copy, modify, and distribute the Archivematica documentation with attribution under the terms of the Creative Commons Attribution Share Alike 4.0 (CC-BY-SA-4.0) license.
+See the [LICENCE](LICENCE) file for details.
+
+# Contributing
+
+Thank you for considering a contribution to the Archivematica documentation!
+For more information on contributing, please see the [Archivematica docs Github wiki](https://github.com/artefactual/archivematica-docs/wiki).
+This wiki describes the change submission process for the Archivematica docs and includes a style guide for new contributions.
+Following these guidelines helps us assess your changes faster and makes it easier for us to merge your submission.
+
+Note that some documentation lives in separate repos:
```

Comment:
Suggest we use repository over its abbreviation for clarity and to reduce the barrier in understanding.

Reply:
Good call.